code/modules/core_implant/cruciform/machinery/obelisk.dm

```diff
@@ -50,9 +55,34 @@
 /obj/machinery/power/nt_obelisk/New()
 	..()
 
+/obj/machinery/power/nt_obelisk/examine(mob/user)
+	..()
+	to_chat(user, "<span class='info'>Level of stored biomatter: [biomatter_ammo]</span>")
+
 /obj/machinery/power/nt_obelisk/attack_hand(mob/user)
 	return
 
+/obj/machinery/power/nt_obelisk/attackby(obj/item/I, mob/user as mob)
+	if (istype(I, /obj/item/stack/material/biomatter))
+		var/obj/item/stack/material/biomatter/B = I
+		if (B.amount)
+			var/sheets_amount_to_transphere = input(user, "How many sheets you want to load?", "Biomatter melting", 1) as num
+			if (sheets_amount_to_transphere > B.amount ) //No cheating!
+				to_chat(user, SPAN_WARNING("You don't that many [B.name]"))
+				return
+			if(sheets_amount_to_transphere)
+				B.use(sheets_amount_to_transphere)
+				biomatter_ammo += sheets_amount_to_transphere
+				user.visible_message(
+									"[user.name] inserted \the [B.name]'s sheets in \the [name].",
+									"You inserted \the [B.name] in  (in amount: [sheets_amount_to_transphere]) \the [name].\
```

Comment:
Wouldn't this return "You inserted _the biomatter sheets in (120 or whatever the ammount you inserted) the Absolutism Obelisk"_ ?
Wouldn't it make sense to put the number of sheets inserted before the name? as in:

> "You inserted \the (in amount: [sheets_amount_to_transphere]) [B.name] in [name].\"

Reply:
This is exstreamly modular for additional content, like perhaps if we make a new oblistic that runs off gold or silver